internal/config/config.go

```diff
@@ -52,11 +52,11 @@ func LoadFromEnvironment(envFiles ...string) (*Config, error) {
 
 	// Database
 	dbConfig := &DatabaseConfig{
-		TimeZone:     os.Getenv(DB_TIMEZONE),
+		TimeZone:     constants.GetOrDefault(os.Getenv(DB_TIMEZONE), constants.DB_DEFAULT_NAME),
```

Comment:
I think you meant `constants.DB_DEFAULT_TIMEZONE` instead of name; either way, once #38 is merged, we can use `getEnvOrDefault` instead.

Reply:
thanks for catching this!